### PR TITLE
#166241001 User should be able to like or dislike and article

### PR DESCRIPTION
**Pull request by deschantkn**
#### What does this PR do?
This PR will allow users to like or dislike articles they view. 

#### Description of Task to be completed?
Have the following endpoints working:
- `/api/articles/like/:articleId`
- `/api/articles/dislike/:articleId`

#### How should this be manually tested?
This task can be tested by running the server and calling the endpoint above with Postman

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#166241001 

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

#### Use Case document for #166241001
https://docs.google.com/document/d/1Ua_TnjGXjfuADeLLdZDSpEnn-oxosLJHw_0vh5oJItU/edit?usp=sharing

### CODE REVIEW

File: src/api/routes/articlesRouter.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 articlesRouter.get('/', getAllArticle);
-
 articlesRouter.get('/:slug', getOneArticle);
-
-articlesRouter.put('/:slug', isAuth.verifyToken, check.articleOwner, validate.updateValidations, updateArticle);
-
-articlesRouter.delete('/:slug', isAuth.verifyToken, check.articleOwner, deleteArticle);
+articlesRouter.put('/:slug', verifyToken, check.articleOwner, validateBody('updateArticle'), updateArticle);
+articlesRouter.delete('/:slug', verifyToken, check.articleOwner, deleteArticle);
+articlesRouter.post('/like/:articleId', verifyToken, likeArticle);
```
Comment (eliemugenzi) [inline src/api/routes/articlesRouter.js]: ```/api/articles/:slug/like```

File: src/api/routes/articlesRouter.js [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
+articlesRouter.post('/dislike/:articleId', verifyToken, dislikeArticle);
```
Comment (eliemugenzi) [inline src/api/routes/articlesRouter.js]: ```/api/articles/:slug/dislike```

**Review by eliemugenzi — APPROVED** [review]

**Review by Diama1 — APPROVED** [review]

**Review by mifeille — APPROVED** [review]

File: src/api/controllers/articlesController.js [hunk 3]
```diff
@@ -108,10 +108,58 @@ class articlesController {
...
+    // Increment like field by one
+    foundArticle.likes += 1;
+
+    // Update table
+    await Article.update(
+      foundArticle,
+      { where: { slug } }
+    );
+
+    return res.status(200).json({ message: 'OK' });
```
Comment (Quantum-35) [inline src/api/controllers/articlesController.js]: Update the response message to something like, `Article Liked.`

**Review by Quantum-35 — CHANGES_REQUESTED** [review]

File: src/api/controllers/articlesController.js (hunk 3 above)
Comment (deschantkn) [reply src/api/controllers/articlesController.js]: Updated.

**Review by Quantum-35 — APPROVED** [review]

**Pull request merged**